vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -584,6 +584,47 @@ testLtsKernel() {
 
 }
 
+testLSMBPF() {
```

Comment:
thi needs to be added to the list of tests that are called.

Reply:
Done